runtime/stdlib/contracts/test.cdc

```diff
@@ -123,6 +123,14 @@ pub contract Test {
                 return self.test(value) || other.test(value)
             })
         }
+
+        /// Returns a new matcher that negates the test of this matcher.
+        ///
+        pub fun not(): Matcher {
```

Comment:
In the original [issue](https://github.com/onflow/cadence/issues/1889), the method signature is:

```cadence
pub fun not(matcher: Matcher): Matcher {
    ...
}
```

However, I don't see how the parameter is of any value here. `not` is a unary operator, unlike the `and` & `or` operators. Correct me If I understood this in the wrong way.

Reply:
I think the idea is that the function is not defined on `Matcher`, but is a global function, so one could say `not(someMatcher)`

Reply:
That makes sense, indeed. However, that would mean it's no longer a matcher combinator. I can make the change easily, I was just wondering what was the desired functionality here.

Reply:
Fixed in https://github.com/onflow/cadence/pull/2420/commits/3b32a0ca3d9850a32cbe1f9d213c0f20ce908580